### PR TITLE
add amip rmse artifact

### PR DESCRIPTION
**Pull request by szy21**
Checklist:
- [x] I created a new folder `$artifact_name`
  - [x] I added a `README.md` in that that folder that
    - [x] describes the data and processing done to it
    - [x] lists the sources of the raw data
    - [x] lists the required citation, licenses
  - [ ] If applicable (e.g., for Creative Commons), I added a `LICENSE` file
  - [x] I added the scripts that retrieve, process, and produce the artifact
  - [x] I added the environment used for such scripts (typically, `Project.toml`
        and `Manifest.toml`)
  - [x] I added the `OutputArtifacts.toml` file containing the information
        needed for package developers to add `$artifact_name` to their package
- [ ] I uploaded the artifact folder to the Caltech cluster (in
      `/groups/esm/ClimaArtifacts/artifacts/$artifact_name`)
- [ ] I added the relevant code to the `Overides.toml` on the Caltech Cluster
      (in `/groups/esm/ClimaArtifacts/artifacts/Overrides.toml`)
- [x] I added a link to the main `README.md` to point to the new artifact

### CODE REVIEW

**Comment by szy21** [comment]
For some reason when I tested it I got `Artifact "cmip_model_rmse" was not found by looking in the paths`. I'll look at it tomorrow.

**Comment by Sbozzolo** [comment]
> For some reason when I tested it I got `Artifact "cmip_model_rmse" was not found by looking in the paths`. I'll look at it tomorrow.

You probably have to instantiate again

File: cmip_model_rmse/OutputArtifacts.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[cmip_model_rmse]
+git-tree-sha1 = "1f6d1c2e5a5fc26c4efd713245ab60ec067bfa73"
```
Comment (Sbozzolo) [inline cmip_model_rmse/OutputArtifacts.toml:2]: What is the total size?

Maybe we want these to be lazy

**Review by Sbozzolo — APPROVED** [review]

File: cmip_model_rmse/OutputArtifacts.toml (hunk 1 above)
Comment (szy21) [reply cmip_model_rmse/OutputArtifacts.toml:2]: It's 4 KB in total

Comment (Sbozzolo) [reply cmip_model_rmse/OutputArtifacts.toml:2]: Okay, it doesn't need to be lazy

**Pull request merged**